.gitignore

```diff
@@ -241,4 +241,7 @@ config.yml
 logs/
 public-key.pem
 locales/
-/cache/
\ No newline at end of file
+
+### VS Code ###
+.vscode
+cache/
```

Comment:
Can you reset this back to `/cache/`? Otherwise it interferes with a folder in Geyser that's called cache.

Reply:
Done